src/renderer/components/virtual-list/virtual-list.tsx

```diff
@@ -13,8 +13,8 @@ import debounce from "lodash/debounce";
 import isEqual from "lodash/isEqual";
 import ResizeSensor from "css-element-queries/src/ResizeSensor";
 
-interface Props<T extends ItemObject = any> {
-  items: T[];
+interface Props {
+  items: any[];
```

Comment:
Maybe this also works? Or even without `| any`

```suggestion
interface Props<T = ItemObject | any> {
  items: T[];
```